Resources/Locale/en-US/objectives/conditions/steal.ftl

```diff
@@ -1,6 +1,11 @@
 objective-condition-steal-title-no-owner = Steal the {$itemName}.
+objective-condition-steal-title-alive-no-owner = Steal {$itemName}.
 objective-condition-steal-title = Steal the {$owner}'s {$itemName}.
 objective-condition-steal-description = We need you to steal {$itemName}. Don't get caught.
 
 objective-condition-steal-station = station
 objective-condition-steal-Ian = head of personnel's corgi
+
+objective-condition-thief-description = The {$itemName} would be a great addition to my collection! 
+objective-condition-thief-animal-description = The {$itemName} would be a great addition to my collection! Most importantly, live.
```

Comment:
```suggestion
objective-condition-thief-animal-description = The {$itemName} would be a great addition to my collection! Most importantly, alive.
```
Upon first reading this, I thought it was saying "get this animal, but most importantly, stay alive" as opposed to "get this animal, but most importantly, make sure you escape with the animal alive"